Makefile

```diff
@@ -118,10 +121,11 @@ fmt: ## Run go fmt against code.
 .PHONY: vet
 vet: ## Run go vet against code.
 	go vet ./...
+	go vet ./... ./api/...
```

Comment:
Doesn't this replace L123?

Reply:
This is an interesting point. I assumed that something like [1] is the way to properly run `go vet` both in the main project and the related `./api` tree.

However, running `go list` like the following:

```
> go list ./... ./api/...
github.com/openstack-k8s-operators/cinder-operator
github.com/openstack-k8s-operators/cinder-operator/controllers
github.com/openstack-k8s-operators/cinder-operator/pkg/cinder
github.com/openstack-k8s-operators/cinder-operator/pkg/cinderapi
github.com/openstack-k8s-operators/cinder-operator/pkg/cinderbackup
github.com/openstack-k8s-operators/cinder-operator/pkg/cinderscheduler
github.com/openstack-k8s-operators/cinder-operator/pkg/cindervolume
github.com/openstack-k8s-operators/cinder-operator/api/v1beta1
```
we can see we're able to "discover" both the main tree of the project, but also the `./api/` part with a single command.
This means that you're actually right, L123 is enough to run `go vet` against the entire project. 
This is also proved by the fact that introducing (by purpose) an error within `cinderapi_types.go` I get:
```
> go vet ./... ./api/...
# github.com/openstack-k8s-operators/cinder-operator/api/v1beta1
api/v1beta1/cinderapi_types.go:132:56: syntax error: unexpected aaa at end of statement
```

So yes, I'll change this part just having L123, thanks @Akrog !

[1] https://github.com/openstack-k8s-operators/glance-operator/blob/master/Makefile#L119-L120